compiler/src/iree/compiler/Modules/HAL/Inline/Transforms/InlineExecutables.cpp

```diff
@@ -132,6 +132,10 @@ class InlineExecutablesPass
       // Build the dispatch function by calling the target function in a loop.
       auto bodyFuncOp =
           innerSymbolTable.lookup<func::FuncOp>(exportOp.getName());
+      if (!bodyFuncOp) {
+        return exportOp.emitOpError("in variant missing body function. Does "
```

Comment:
just saying missing function is enough here - the note about supporting or not is out of place as this error will happen if the user doesn't put a function in, or the function has any other type besides func::FuncOp, etc and seeing this question in such a case would be weird

if we want to properly bail then we could add a check around https://github.com/openxla/iree/blob/b47ed8c93d82b2030db4592f57a78afd470dbc08/compiler/src/iree/compiler/Pipelines/Pipelines.cpp#L252-L255 that ensures the executable target is vmvx-inline and be more explicit about the failure

Reply:
I didn't see a way to emit an error in `void buildIREEVMTransformPassPipeline()`, so added code up a level in Embed.cpp.